scripts/package.json

```diff
@@ -77,6 +77,7 @@
     "request-promise-native": "^1.0.5",
     "resolve": "^1.7.1",
     "riceburn": "^1.3.1",
+    "semver": "^7.3.5",
```

Comment:
semver is  already adhering to single version policy for dev dependencies -> installed globally in root package.json -> please remove